experimental/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -138,23 +105,17 @@ export class RootDataObject
         } else if (isSharedObjectClass(objectClass)) {
             return this.createSharedObject<T>(objectClass);
         }
-
         throw new Error("Could not create new Fluid object because an unknown object was passed");
     }
 
-    public async getDataObject<T extends IFluidLoadable>(id: string) {
-        const handle = await this.root.wait<IFluidHandle<T>>(id);
-        return handle.get();
-    }
-
-    private async createDataObject<T extends IFluidLoadable>(dataObjectClass: DataObjectClass<T>): Promise<T> {
+    public async createDataObject<T extends IFluidLoadable>(dataObjectClass: DataObjectClass<T>): Promise<T> {
```

Comment:
The `create` call in line 100-109 already exposes this functionality without requiring the developer to distinguish between data object and shared object. Why are we additionally also exposing these two functions?

Reply:
+1 . The caller does not need to care about DO vs DDS. This is RootDataObject logic

Reply:
yeah, not sure why those were public. changing to private